phase1-coordinator/src/objects/round.rs

```diff
@@ -1010,10 +1010,8 @@ impl Round {
                     }
                 }
 
-                // Remove the given contribution and all subsequent contributions.
-                for contribution_id in task.contribution_id()..(chunk.get_contributions().len() as u64) {
-                    chunk.remove_contribution_unsafe(contribution_id);
-                }
+                // Remove the given contribution
+                chunk.remove_contribution_unsafe(task.contribution_id());
```

Comment:
Dependent contributions are now specified to be removed at a higher level in the code (`affected_tasks` in `coordinator_state.rs`). This was causing the inconsistency between the round state and the actual files on disk by removing the contributions from the round state without removing them from disk.